deploybintray.json

```diff
@@ -11,7 +11,7 @@
 
     "files":
         [
-        {"includePattern": "target/(.*)", "uploadPattern":"/$1"}
+        {"includePattern": "$TRAVIS_BUILD_DIR/package/(.*)", "uploadPattern":"/$1"}
```

Comment:
This was a dumb copy/paste mistake. There is no folder called "target" anywhere in the build tree.